.github/workflows/rpe_test.yml

```diff
@@ -81,6 +81,11 @@ jobs:
         if: ${{ matrix.os == 'ubuntu-latest' }}
         run: |
             npx eslint src/
+            
+      - name: Run frontend unit test only on ubuntu-latest
+        if: ${{ matrix.os == 'ubuntu-latest' }}
```

Comment:
Why not other OSes ? Is this temporary?
FYI: @NadeemYaseen

Reply:
I did the same as for eslint, but it make sense to have unit tests for all OS. Fixed